bodhi/server/scripts/approve_testing.py

```diff
@@ -96,19 +95,8 @@ def main(argv=sys.argv):
                     config.get('testing_approval_msg') % update.release.mandatory_days_in_testing)
                 update.comment(db, text, author=u'bodhi')
 
-
-def _get_db_session(config_uri):
-    """
-    Construct and return a database session using settings from the given config_uri.
-
-    :param config_uri: A path to a config file to use to get the db settings.
-    :type  config_uri: basestring
-    :return:           A database session
-    """
-    # There are many blocks of code like this in the codebase. We should consolidate them into a
-    # single utility function as described in https://github.com/fedora-infra/bodhi/issues/1028
-    settings = get_appsettings(config_uri)
-    engine = engine_from_config(settings, 'sqlalchemy.')
-    Session = scoped_session(sessionmaker(extension=ZopeTransactionExtension()))
-    Session.configure(bind=engine)
-    return Session()
+        db.commit()
+    except SQLAlchemyError as e:
```

Comment:
I notice that in the untag branched script you caught the general exception to rollback, but here and in expire overrides you catch only this one class of exceptions. It might be good to be consistent.